pages/docs/tutorials/app-example-config.mdx

```diff
@@ -0,0 +1,672 @@
+---
+title: "App Example: Declarative Sync"
+date: "2020-03-30"
+pullQuote: "Learn how to sync data from a SQLite customer database to a Mailchimp audience."
+pageNavSelector: "h2"
+---
+
+In this tutorial we're going to look at how we built our [app-example-config project](https://github.com/grouparoo/app-example-config). We'll start from scratch (with some seeded data) and sync our customer data from a SQLite database Source to a Mailchimp Destination.
+
+## Step 1: Install Grouparoo
+
+First, make sure you have the [Grouparoo CLI tool](/docs/cli) installed.
+
+    $ npm install -g grouparoo
+
+You can check to make sure it worked by checking the version.
+
+    $ grouparoo --version
+
+## Step 2: Create Project
+
+Next, make a new directory for your Grouparoo project, then run the `init` command. Let's say we wanted our project to be called `app-example-config`. We might do something like this:
+
+    $ cd /path/to/workspace
+    $ grouparoo init app-example-config
+    $ cd app-example-config
+
+_Note: If the directory for your project exists, you can run `grouparoo init .` and Grouparoo will create the project inside that directory._
+
+## Step 3: Add Source Database
+
+We have some seed data for you to work with for this example. You can download the database directly [here](/downloads/sqlite-source.sqlite). Place that file anywhere on your file system that your current user can read.
```

Comment:
... This SQLite database contains the contents of this CSV [link], which is a prototypical ecommerce example.